lib/transactions/transfer.js

```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating dapp transfer in Lisk Blockchain
```

Comment:
Transfer module provides functions for use with DAPP (blockchain apps?) transfers to the Lisk Blockchain.